### PR TITLE
feat(network): Calico upgrade

### PR DESCRIPTION
**Pull request by angao**
**What type of PR is this?**
/kind upgrade
/area network

**What this PR does / why we need it**:
- upgrade calico to v3.17.0
- wait this [pr](https://github.com/caicloud/kubespray/pull/39)

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @WangMinjie114 @hanxueluo @bbbmj 

**Does this PR introduce a user-facing change?**:

```release-note
Upgrade calico to v3.17.0
```

### CODE REVIEW

File: roles/network_plugin/bd-calico/templates/calico-node.yml.j2 [hunk 1]
```diff
@@ -64,7 +62,7 @@ spec:
         # and CNI network config file on each node.
         - name: install-cni
           image: {{ calico_cni_image_repo }}:{{ calico_cni_image_tag }}
-          command: ["/install-cni.sh"]
+          command: ["/opt/cni/bin/install"]
```
Comment (hanxueluo) [inline roles/network_plugin/bd-calico/templates/calico-node.yml.j2:65]: 上游做了修改？

Comment (angao) [reply roles/network_plugin/bd-calico/templates/calico-node.yml.j2:65]: 是的，我把部署好的 calico，直接换成 v3.17.0 镜像测试的

**Pull request merged**